### ISSUE TITLE
Keyboard shortcut cursor sometimes feels unresponsive 

### ISSUE THREAD

**Issue by thomasballinger** [google/blockly#3225]
**Describe the bug**

When "accessibility mode" is turned on at demos/keyboard_nav/index.html, the cursor animate fades in out similar to the blinking caret in a text input. However unlike a text input blinking caret, the fading animation in Blockly does not reset when the cursor moves. Because of this, the effect of keypresses that occur while the cursor is not visible appears delayed to the user.

**To Reproduce**

1. Open demos/keyboard_nav/index.html in a browser
2. Click the 'Enable Accessibility Mode' checkbox
3. Press 'd' or 's' to move the cursor
4. Try moving the cursor while the cursor is not visible

**Expected behavior**

Like a text edit caret, the cursor should immediately become visible when it moves.

**Screenshots**

![fixhighlight](https://user-images.githubusercontent.com/458879/66717766-9f4a5380-ed91-11e9-918b-f78dc47e1d1f.gif)

**Desktop (please complete the following information):**
 - OS: OSX 10.14
 - Browser: Firefox + Chrome
 - Version: Firefox Nightly 71.0a1 (2019-10-02) + Chrome Version 77.0.3865.90 (Official Build) (64-bit)

**Additional context**

I discussed this in-person with Abby at the Blockly Summit Hackathon on Oct 11.

**Comment by samelhusseini** [comment]
Fixed.
